spec/system/eligibility_spec.rb

```diff
@@ -275,13 +276,8 @@ def then_i_do_not_see_the_start_page
   end
 
   def then_i_see_the_start_page
-    expect(page).to have_content(
-      "Check your eligilibity to apply for qualified teacher status (QTS) in England"
-    )
-    expect(page).to have_current_path("/eligibility/start")
-    expect(page).to have_title(
-      "Check your eligilibity to apply for qualified teacher status (QTS) in England"
-    )
+    expect
+    start_page.heading.text.to eq "Check your eligibility to apply for qualified teacher status (QTS) in England"
```

Comment:
```suggestion
    expect(start_page.heading.text).to eq("Check your eligibility to apply for qualified teacher status (QTS) in England")
```

Looks like the `expect` is across multiple lines so I'm not sure it would actually do the expectation.